### PR TITLE
update /download/server/thank-you to vbt

### LINKED ISSUE #1646: /download/server/thank-you/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* updated /download/server/thank-you to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/download/server/thank-you) and [demo](http://www.ubuntu.com-1646-download-server-thank-you.demo.haus/download/server/thank-you)
- compare to the [design](https://www.dropbox.com/search/work?path=%2F00_web_team%2F_ubuntu.com%2Fnew+projects%2Fvanilla+update%2Fpatterns+inventory%2F9.+Download&preview=download-server-thank-you.png&qsid=08186058431934915572202485356221&query=server-thank-you)

## Issue / Card

Fixes #1646

### CODE REVIEW

**Comment by yaili** [comment]
- [ ] In the original version in the screenshots there is a sub-heading, but in the live version there isn't anymore. Is this on purpose?
- [ ] Get MAAS should have button too
- [ ] The external icon should match the text colour in button - where should this be fixed?
- [ ] There doesn't feel like there should be a divider between Landscape and MAAS as they're both part of "Your next steps with Ubuntu Server" - can we remove it?

**Comment by pmahnke** [comment]
@yaili updated

* '... there is a sub-heading...' only if you select a version and architecture... I added an error message so you can see
* the 'external icon' isn't part of the pattern, it is a span, so I guess vbt for now... but I fixed it in ubuntu
* should we add the smile to all thank-yous? and perhaps an orange one?

**Comment by yaili** [comment]
Getting an error in this demo.

File: templates/download/_nav_tertiary-v1.html [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
   {% endif %}
   {% if level_2 == 'cloud' %}
-    <li><a {% if level_3 == 'conjure-up' %} class="is-active"{% endif %}href="/download/cloud/conjure-up">Conjure-up</a></li>
-    <li><a {% if level_3 == 'autopilot' %} class="is-active"{% endif %}href="/download/cloud/autopilot">Autopilot</a></li>
+    <li class="p-inline-list__item"><a class="p-inline-list__link {% if level_3 == 'conjure-up' %}is-active{% endif %}"href="/download/cloud/conjure-up">Conjure-up</a></li>
+    <li class="p-inline-list__item"><a class="p-inline-list__link {% if level_3 == 'autopilot' %}is-active{% endif %}"href="/download/cloud/autopilot">Autopilot</a></li>
   {% endif %}
   {% if level_3 == 'burn-a-dvd-on-macos' %}
-    <li><a class="is-active" href="/download/desktop/burn-a-dvd-on-macos">How to burn a DVD on macOS</a></li>
+    <li class="p-inline-list__item"><a class="p-inline-list__link class="is-active" href="/download/desktop/burn-a-dvd-on-macos">How to burn a DVD on macOS</a></li>
```
Comment (barrymcgee) [inline templates/download/_nav_tertiary-v1.html]: Double class declarations

**Comment by yaili** [comment]
@pmahnke this demo still not working

**Comment by barrymcgee** [comment]
@yaili @pmahnke This was due to failing Sass build, I've just fixed it and kicked the build.

**Comment by yaili** [comment]
@pmahnke yes I think we should add a smile to all thank-yous, I think I added this in my Dropbox notes, or at least in many of them

File: templates/download/_nav_tertiary-v1.html (hunk 1 above)
Comment (pmahnke) [inline templates/download/_nav_tertiary-v1.html]: fixed @barrymcgee

**Pull request merged**